physics/GWD/ugwpv1_gsldrag.F90

```diff
@@ -63,6 +63,7 @@ module ugwpv1_gsldrag
 !>@brief The subroutine initializes the unified UGWP
 !> \section arg_table_ugwpv1_gsldrag_init Argument Table
 !! \htmlinclude ugwpv1_gsldrag_init.html
+!!
```

Comment:
@mzhangw We need to keep an empty line with `!!` after the `htmlinclude` line in order for ccpp_prebuild to properly parse the file.